src/components/Form/Picklist.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import enhanceWithClickOutside from 'react-click-outside';
+import { memoize } from 'lodash';
```

Comment:
Importing from `lodash/memoize` is better here since `{} from 'lodash'` does not work with tree shaking for webpack<4 and requires `babel-plugin-lodash` to be present to reduce build size when requiring `react-lds`